test/support/boot_requirements_context.rb

```diff
@@ -29,57 +29,22 @@ def find_vol(mount_point, volumes)
     volumes.find { |p| p.mount_point == mount_point }
   end
 
-  subject(:checker) { described_class.new(devicegraph) }
```

Comment:
We should not touch this context because it affects to boot requirements tests, which are used for documenting booting requirements. Probably this is too sensitive to be modified at this moment. I would keep it untouched.

Moreover, I would not use it in `boot_requirements_errors_test.rb`. Instead, I would directly add what is necessary for those tests instead of using this context.

Reply:
well, plan is to remove overmocking also in boot_requirements test to operate on real device graph.